File: Model.Tests/SetResultTests.vb
<TestClass()> _
Public Class SetResultTests

    <TestMethod()> _
    Public Sub ToStringTest()
        ' Arrange
        Dim setResult = New SetResult() With {.Home = 5, .Guest = 3}

        ' Act
        Dim result = setResult.ToString()

        ' Assert
        Assert.AreEqual("5:3", result, "Unexpected result")
    End Sub

End Class

File: POFF.Kicker.Tests/SetResultTests.cs
using NUnit.Framework;
using POFF.Kicker.Domain;

namespace POFF.Kicker.Tests;

[TestFixture]
class SetResultTests
{
    [Test]
    public void ToStringTest()
    {
        // Arrange
        var setResult = new SetResult() { Home = 5, Guest = 3 };

        // Act
        string result = setResult.ToString();

        // Assert
        Assert.That(result, Is.EqualTo("5:3"), "Unexpected result");
    }
}